cmd/admission-webhook-server/Dockerfile

```
@@ -0,0 +1,15 @@
+FROM golang as builder
+
+WORKDIR /usr/src/lokomotive-admission-webhook-server
```

As we build using `lokomotive` repository/source code:
```suggestion
WORKDIR /usr/src/lokomotive
```